### PR TITLE
Fix multi-machine test in CC system role

### PR DESCRIPTION
**Pull request by Amrysliu**
Please see individual commit message.

- Related ticket: https://progress.opensuse.org/issues/102116
- Needles: N/A
- Verification run: 
         https://openqa.suse.de/tests/7635686 (create qcow2 file in x86_64)
         https://openqa.suse.de/tests/7635803 (create qcow2 file in aarch64)
         https://openqa.suse.de/tests/7635869 (multi-machine test in x86_64)
         https://openqa.suse.de/tests/7636180 (multi-machine test in aarch64)
         https://openqa.suse.de/tests/7647288 (selinux_setup passed in s390x )

### CODE REVIEW

File: tests/security/cc/remove_bridge_network.pm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    assert_script_run('rm -f /etc/sysconfig/network/ifcfg-br0');
+    assert_script_run('sed -i \'s/none/dhcp/\' /etc/sysconfig/network/ifcfg-eth0');
```
Comment (rfan1) [inline tests/security/cc/remove_bridge_network.pm:22]: Xiaojing, 

It is not good way to use hardcode "eth0". please use "bridge" cmd to get the right NIC name and bridge name.

File: tests/security/cc/remove_bridge_network.pm [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    my $bridge_info = script_output('bridge link');
+    if ($bridge_info =~ /\d+:\s+(eth\d):.*master\s(br\d).*/) {
```
Comment (rfan1) [inline tests/security/cc/remove_bridge_network.pm:22]: xiaojing,
Please do not use eth here. the name may change as udev rules [emx enpx etc]

Comment (Amrysliu) [reply tests/security/cc/remove_bridge_network.pm]: Thanks, fixed.

**Review by lilyeyes — APPROVED** [review]
LGTM.

File: schedule/security/create_hdd_common_criteria_for_mm.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: create_hdd_common_criteria_for_mm
+description: >
+    This is used to publish the qcow2 file which is CC system role without
+    bridge network and all packages are needed have been installed.
```
Comment (rfan1) [inline schedule/security/create_hdd_common_criteria_for_mm.yaml:4]: bridge network and all packages are needed have been installed. -> bridge network,  required packages for mm setup usage have been installed as well.

File: tests/security/cc/remove_bridge_network.pm [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
```
Comment (jouyingbin) [inline tests/security/cc/remove_bridge_network.pm:18]: tiny comment: looks like one extra blank line here.  :)

File: schedule/security/create_hdd_common_criteria_for_mm.yaml (hunk 3 above)
Comment (Amrysliu) [reply schedule/security/create_hdd_common_criteria_for_mm.yaml]: Thanks, fixed

File: tests/security/cc/remove_bridge_network.pm (hunk 4 above)
Comment (Amrysliu) [reply tests/security/cc/remove_bridge_network.pm]: Thanks, fixed

**Review by jouyingbin — APPROVED** [review]
LGTM. Thanks for the update.

**Review by rfan1 — APPROVED** [review]
LGTM

**Pull request merged**